wuvt/templates/admin/donation_index.html

```diff
@@ -62,7 +67,12 @@ <h1>Donations</h1>
                 {{ donation.city }}, {{ donation.state }} {{ donation.zipcode }}
 {% endif %}
             </td>
-            <td>{% if donation.shipped_date %}{{ donation.shipped_date|datetime }} {% endif %}</td>
+            <td>{% if donation.shipped_date %}{{ donation.shipped_date|datetime }}
+		{% else %}<form method="post">
+			<input type="hidden" name="_csrf_token" value="{{ csrf_token() }}">
+			<input type="hidden" name="id" value="{{ donation.id }}">
+			<button name="set_shipped" type="submit" class="btn btn-sm">Set as Shipped</button></form>
+		{% endif %}{% endif %}</td>
```

Comment:
I'm assuming this is a typo. This is causing the following error:
```
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endif'. Jinja was looking for the following tags: 'endfor' or 'else'. The innermost block that needs to be closed is 'for'.
```

Reply:
I also made this change and the button seems to still be showing after I click. Not sure if it's not updating the database or not but it doesn't work.